src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,81 @@
+import { Icon } from "@/components";
+import * as RadixCheckbox from "@radix-ui/react-checkbox";
+import styled from "styled-components";
+
+export type CheckboxProps = {
+  label?: string;
+  isDisabled?: boolean;
+  isChecked?: boolean;
+  onCheckedChange?: (checked: boolean) => void;
+};
+export const Checkbox = ({
+  label = "",
+  isDisabled = false,
+  isChecked = false,
+  onCheckedChange,
```

Comment:
can we just standardise to a simple `onChange`? It would follow the principle of least surprise when using input family components.

Reply:
i kept the original action, but i agree on having a standard for all components

Reply:
done